packages/k/keyd/package.yml

```
@@ -0,0 +1,22 @@
+name       : keyd
+version    : 2.5.0
+release    : 1
+source     :
+    - https://github.com/rvaiya/keyd/archive/refs/tags/v2.5.0.tar.gz : 93ec6c153ef673a7a8b4d8b686494dee11d182513f4531c71dce15a8db7f6c1c
+homepage   : https://github.com/rvaiya/keyd
+license    : MIT
+component  : system.utils
+summary    : A key remapping daemon for linux.
+description: |
+    keyd provides a flexible system wide daemon which remaps keys using kernel level input primitives (evdev, uinput).
+build      : |
+    %make
+install    : |
+    %make_install PREFIX=/usr SOCKET_PATH=/run/keyd.socket
+    sed -e 's#@PREFIX@#/usr#' -e 's#multi-user.target#systemd-udev.service#' keyd.service.in > keyd.service
+    install -Dm00644 keyd.service -t $installdir/usr/lib/systemd/system/
+
+    # Start service by default
+    install -dm00755 $installdir/usr/lib/systemd/system/multi-user.target.wants
+    ln -sv ../keyd.service $installdir/usr/lib/systemd/system/multi-user.target.wants/keyd.service
+    rm -r $installdir/etc/
```

We like to see what's happening.
```suggestion
    rm -rv $installdir/etc/
```